berkeley-function-call-leaderboard/bfcl/model_handler/api_inference/nova.py

```diff
@@ -74,6 +75,11 @@ def _query_FC(self, inference_data: dict):
         return api_response, end_time - start_time
 
     def _pre_query_processing_FC(self, inference_data: dict, test_entry: dict) -> dict:
+        for round_idx in range(len(test_entry["question"])):
+            test_entry["question"][round_idx] = combine_consecutive_user_prompts(
+                test_entry["question"][round_idx]
```

Comment:
@HuanzhiMao if the documentation links it we should support this. But for reference, can you also link the documentation in the code as a simple python comment in line 78?

Reply:
There is no documentation link, but their endpoint will throw this error:
```
An error occurred (ValidationException) when calling the Converse operation: A conversation must alternate between user and assistant roles. Make sure the conversation alternates between user and assistant roles and try again.
```